vm/src/primitives.rs

```diff
@@ -284,6 +364,26 @@ pub fn load(vm: &Thread) -> Result<()> {
         show_Int => primitive!(1 prim::show_int),
         show_Float => primitive!(1 prim::show_float),
         show_Char => primitive!(1 prim::show_char)
+        add_Int => primitive!(2 add_int),
```

Comment:
Why not `s/add_int/VMInt::add`? I guess the name in the vm would be different but maybe we should overload `primitive!` to allow a different name?